### PR TITLE
Fixes #33430 - do not error on pulpcore with 1 worker

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
easy to test, shut down all but 1 worker and try to use pulp3

File: app/models/katello/ping.rb [hunk 1]
```diff
@@ -195,10 +195,7 @@ def pulp3_without_auth(url)
         end
 
         workers = json["online_workers"] || []
-
-        unless workers.count > 1
-          fail _("Not all necessary pulp workers running at %s.") % url
-        end
+        fail _("Not all necessary pulp workers running at %s.") % url if workers.empty?
```
Comment (rverdile) [inline app/models/katello/ping.rb:198]: Could the error message be "No pulp workers running..." if it's failing when workers is empty?

Comment (jlsherrill) [reply app/models/katello/ping.rb]: updated!

**Review by jjeffers — APPROVED** [review]
Test all 3 conditions - >1, 1, and 0 pulp workers - worked as indicated. 

If there are no pulp workers, the error message appears. 1 or more pulp workers allows the operation to complete.

**Pull request merged**